### PR TITLE
[Scouting] Improve readability and efficiency

### PR DESCRIPTION
**Pull request by strahinja-misic**
Small contribution to the project

### CODE REVIEW

**Comment by strahinja-misic** [comment]
@marcus-j Trying to figure out what is going on. 'mvn clean verify' runs without any issues locally for both my branch and master but if I run the test method individually it fails. Also, test itself has nothing to do with my changes...

**Comment by marcus-j** [comment]
@strahinja-misic I believe we saw this previously happen as well. I also believe that your changes do not cause this. Still, if you have time, feel free to look into it :)

**Review by roechi — APPROVED** [review]
Looks great! Thanks for your engagement!

**Comment by strahinja-misic** [comment]
@marcus-j Could you take a look at this PR when you have time?

**Comment by marcus-j** [comment]
> @marcus-j Could you take a look at this PR when you have time?

@strahinja-misic I did leave two comments a while back (see above):

image

**Comment by strahinja-misic** [comment]
@marcus-j In case you have written additional comments I just wanted to say that I do not see any for some reason.

**Review by marcus-j — CHANGES_REQUESTED** [review]

**Comment by marcus-j** [comment]
> @marcus-j In case you have written additional comments I just wanted to say that I do not see any for some reason.

Damn, forgot to finish that review and GH hides the comments that have been made during that :/ Sorry!

File: src/main/java/de/idealo/kafka/deckard/proxy/ProducerProxyBeanFactory.java [hunk 1]
```diff
@@ -108,17 +108,17 @@ public ProducerProxyBeanFactory(
         }
 
         private boolean isValidEncryptionSetup(String password, String salt) {
-            return !StringUtils.isEmpty(password) && !StringUtils.isEmpty(salt);
+            return !password.isEmpty() && !salt.isEmpty();
```
[the remark this answers is not in the document]
Comment (strahinja-misic) [reply src/main/java/de/idealo/kafka/deckard/proxy/ProducerProxyBeanFactory.java]: Fixed.

File: src/main/java/de/idealo/kafka/deckard/proxy/ProducerProxyBeanFactory.java [hunk 2]
```diff
@@ -94,7 +94,7 @@ public ProducerProxyBeanFactory(
         }
 
         private Serializer<V> encryptedIfConfigured(KafkaProducer kafkaProducer, Serializer<V> embeddedSerializer) {
-            if (!kafkaProducer.encryptionPassword().equals("") || !kafkaProducer.encryptionSalt().equals("")) {
+            if (!kafkaProducer.encryptionPassword().isEmpty() || !kafkaProducer.encryptionSalt().isEmpty()) {
```
[the remark this answers is not in the document]
Comment (strahinja-misic) [reply src/main/java/de/idealo/kafka/deckard/proxy/ProducerProxyBeanFactory.java]: Fixed.

**Review by marcus-j — APPROVED** [review]

**Pull request merged**